package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "qpp-measures-data",
-  "version": "1.22.1",
+  "version": "1.22.3",
```

Comment:
What happened to 1.22.2?

Reply:
I have another PR open. Whichever gets merged first will be incremented correctly. I  only incremented this to `1.22.3` because I already incremented the other PR as `1.22.2`

Reply:
I have addressed this and followed the process.